source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -311,7 +343,7 @@ Node Details
     - Details
 
   * - Menu
-    - Can have 1‐9 child nodes, with each child corresponding to the 1‐9 buttons. At the end of the child list, it can also optionally have failover child nodes designated by a node with a keypress_failover_type parameter. See example below.
+    - Can have 1‐9 child nodes, with each child corresponding to the 1‐9 buttons. At the end of the child list, it can also optionally have failover child nodes designated by a node with a keypress_failover_type parameter (see example below). If speech recognition is enabled, the caller may also respond verbally with their menu choice by saying the designated phrase.
```

Comment:
Given that currently it only supports verbally saying the digit, I'd suggest only sharing that for now and then we can update to this when ASR custom grammars is available.

> If speech recognition is enabled, the caller may also verbally say the digit of their choice.

Reply:
Gotcha, I will make that update. Also heh, sorry you were too fast!  I was editing the screenshots when you reviewed. They've been posted in the ticket's comments, on this PR, and in #dev-sequoia-demos.

Reply:
How does this sound? ```
If speech recognition is enabled, the caller may also respond verbally with their menu choice (i.e. "one" for 1, and "two" for 2)
```